### PR TITLE
Version 1.3.0

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: .drone-local.yml [hunk 1]
```diff
@@ -68,7 +69,7 @@ services:
 matrix:
   include:
     - {
-        GO: "golang:1.8", DATABASE: postgres, VERSION: 9.6, REFORM_DRIVER: postgres,
+        GO: "golang:1.9", DATABASE: postgres, VERSION: 9.6, REFORM_DRIVER: postgres,
```
Comment (rumyantseva) [inline .drone-local.yml]: Maybe it is possible to use some pre-defined variables for this Go version to not to change a lot of similar lines each time?

File: README.md [hunk 2]
```diff
@@ -5,26 +5,43 @@
 [![Travis CI Build Status](https://travis-ci.org/go-reform/reform.svg?branch=v1-stable)](https://travis-ci.org/go-reform/reform)
 [![Drone CI Build Status](https://drone.aleksi.io/api/badges/go-reform/reform/status.svg)](https://drone.aleksi.io/go-reform/reform)
 [![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/kbkyjmic461xa7b3/branch/v1-stable?svg=true)](https://ci.appveyor.com/project/AlekSi/reform/branch/v1-stable)
-[![Coverage Status](https://coveralls.io/repos/github/go-reform/reform/badge.svg?branch=v1-stable)](https://coveralls.io/github/go-reform/reform?branch=v1-stable)
+[![Coverage Report](https://codecov.io/gh/go-reform/reform/branch/v1-stable/graph/badge.svg)](https://codecov.io/gh/go-reform/reform)
 [![Go Report Card](https://goreportcard.com/badge/gopkg.in/reform.v1)](https://goreportcard.com/report/gopkg.in/reform.v1)
 
+<a href="https://en.wikipedia.org/wiki/Peter_the_Great"><img align="right" alt="Reform gopher logo" title="Peter the Reformer" src=".github/reform.png"></a>
```
Comment (rumyantseva) [inline README.md]: Is it possible to use HTML here? Why don't you use markdown? Because of `align="right"`?

Comment (AlekSi) [reply README.md]: For all attributes.

File: .drone-local.yml (hunk 1 above)
Comment (AlekSi) [reply .drone-local.yml]: That file is a subset of .drone.yml. I just copy/paste a part of it. Making a variable will make it harder.

**Pull request merged**